app/res/values-de/strings.xml

```diff
@@ -252,8 +252,8 @@
     <string name="code">Code</string>
     <string name="following_user">Folgen…</string>
     <string name="unfollowing_user">Entfolgen…</string>
-    <string name="starring_repository">Hervorheben…</string>
-    <string name="unstarring_repository">Hervorhebung entfernen…</string>
+    <string name="starring_repository">Favorisieren…</string>
```

Comment:
There should be a difference between the translations of ```star```and ```starring_repository```

(Same between ```unstar``` and ```unstarring_repository```)